src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/_client_factory.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+def cognitiveservices_client_factory(_):
+    from azure.cli.core.commands.client_factory import get_mgmt_service_client
+    from azure.mgmt.cognitiveservices import CognitiveServicesManagementClient
+    return get_mgmt_service_client(CognitiveServicesManagementClient, location='notused').cognitive_services_accounts
```

Comment:
there is an extra parameter need passing a value when initializing the sdk management client. but it's not used by every class and operation. so for some initialization, i need pass a dummy value.